features/donation.feature

```diff
@@ -53,17 +53,17 @@ Feature: The goal is to donate one time or multiple time with a subscription
     And I click the "donation_check_label" element
     And I click the "donation_check_nationality_label" element
     And I press "Je donne"
-    Then I wait 5 second until I see "Numéro de carte"
```

Comment:
If this is removed, do we still need to use Selenium ?

Reply:
@B-Galati, Selenium is needed to initialize the donation request by going in `"/don"` page before each scenario which has the react component.